nimble_build_system/cad/shelf.py

```diff
@@ -27,13 +28,12 @@
 from nimble_build_system.orchestration.paths import REL_MECH_DIR
 
 
-def create_shelf_for(device_id: str,
-                     *,
-                     assembly_key: str='Shelf',
-                     position: tuple[float, float, float]=(0,0,0),
-                     color: str='dodgerblue1',
-                     rack_params: RackParameters|None = None,
-                     dummy_device_data:dict|None=None):
+def create_shelf_for_x(device_id: str,
```

Comment:
Is there a reason for this name change? I felt that `create_device_for("NUC_1234")` is pretty clear?